bare-metal-2.ks.in

```diff
@@ -0,0 +1,54 @@
+#version=DEVEL
+#test name: bare-metal-2
+# NOTE: This is not a supported use case, but it is easier to test.
```

Comment:
Please elaborate a bit more on this. What is the use case here?

Reply:
Fixed.